pypyr-nodejs/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM pypyr/pypyr:1.1.1
+
+ENV NODE_VERSION 10.13.0
+ARG NPM_VERSION=6.4.1
+ARG YARN_VERSION=1.10.1
+
+USER root
+RUN apt-get update && apt-get install -y sudo
+
+# Install Node.js
+RUN groupadd --gid 1001 node \
+  && useradd --uid 1001 --gid node --shell /bin/bash --create-home node
```

Comment:
I know I gave you this, but looking at it, I'm not all that convinced it the user/group _node_ actually gets used. . . does anything break if you take this out? sorry about that!